src/annotator/config.js

```diff
@@ -47,7 +47,7 @@ function configFrom(window_) {
   //
   // In environments where the config has not been injected into the DOM,
   // we try to retrieve it from the URL here.
-  var directLinkedID = annotationQuery.extractAnnotationQuery(window_.location.href);
+  var directLinkedID = extractAnnotationQuery(window_.location.href);
```

Comment:
By the way is this setting called "annotation query" or is it "direct linked ID" (or is "annotation ID" or "annotation IDs" as it also gets called elsewhere)? I'll tidy up the naming of this setting in a later PR